plugins/package-managers/python/src/main/kotlin/Pip.kt

```diff
@@ -39,8 +39,8 @@ import org.ossreviewtoolkit.utils.ort.showStackTrace
 private const val OPTION_OPERATING_SYSTEM_DEFAULT = "linux"
 private val OPERATING_SYSTEMS = listOf(OPTION_OPERATING_SYSTEM_DEFAULT, "macos", "windows")
 
-private const val OPTION_PYTHON_VERSION_DEFAULT = "3.10"
-private val PYTHON_VERSIONS = listOf("2.7", "3.6", "3.7", "3.8", "3.9", OPTION_PYTHON_VERSION_DEFAULT)
+private const val OPTION_PYTHON_VERSION_DEFAULT = "3.11"
```

Comment:
We cannot really make this the default version already in this commit, if only the next commit is adding that version.

Reply:
To fix this, I changed the order of the commits.

Reply:
To fix this finally I have squashed all commits to one.